### PR TITLE
[EPMEDU-1270]: Add html markup support for feeding points description

### PR DESCRIPTION
**Pull request by Andrii-Chornyi**
Done by using library https://github.com/ireward/compose-html

<details>
  <summary> Default Text: </summary>

result before

</details>

<details>
  <summary> MarkupText: </summary>

result after

</details>

<details>
  <summary> Real-case example: </summary>

backend

backend after

</details>
Currently supported tags:<br>
- <`br> <br>
- <`p> <br>
- <`h1>-<`h6> <br>
- <`b> <br>
- <`s>  <br>
- <`i> <br>
-  <`br> <br>
-<`u> <br>
- ordered and unordered lists- <br>
- links by <`a>, but they are not clickable. <br>

Currently, links are not clickable.

### CODE REVIEW

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/repository/FeedingPointRepositoryImpl.kt [hunk 1]
```diff
@@ -20,9 +20,14 @@ class FeedingPointRepositoryImpl : FeedingPointRepository {
         FeedingPoint(
             index,
             title = "$index - Near to Bukia Garden M.S Technical University",
-            "$index - This area covers about 100 sq.m. -S, it starts with Bukia Garden " +
-                "and Sports At the palace. There are about 1000 homeless people here " +
-                "The dog lives with the habit of helping You need.",
+            "Text outside markuptags. <h1> Header1 </h1> <h2> Header2 </h2> <h3> Header3 </h3>" +
```
Comment (Fileek) [inline shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/repository/FeedingPointRepositoryImpl.kt]: Please rebase onto develop branch and put this text in preview

**Review by Fileek — CHANGES_REQUESTED** [review]

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/repository/FeedingPointRepositoryImpl.kt (hunk 1 above)
Comment (Andrii-Chornyi) [reply shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/repository/FeedingPointRepositoryImpl.kt]: Rebased

**Comment by Andrii-Chornyi** [comment]
I decided to drop using DailyTags library due to its poor support of nested tags and some other minor issues. Task was redone to use a new library HtmlText which relies on android's default behavior in form of Html.FromHtml method.

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/markuptext/MarkupText.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.epmedu.animeal.foundation.markuptext
```
Comment (Fileek) [inline library/foundation/src/main/java/com/epmedu/animeal/foundation/markuptext/MarkupText.kt]: Can we rename the `markuptext` package to just a `text`?

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/markuptext/OrderedListParser.kt [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.epmedu.animeal.foundation.markuptext
```
Comment (Fileek) [inline library/foundation/src/main/java/com/epmedu/animeal/foundation/markuptext/OrderedListParser.kt]: Can we put both parsers into a separate sub-package (e.g. `com.epmedu.animeal.foundation.text.parser`)?

**Review by Fileek — COMMENTED** [review]
Good job! 
I have only 2 more minor comments regarding packages

**Review by Fileek — APPROVED** [review]

**Pull request merged**